src/AppBundle/Entity/Post.php

```diff
@@ -80,10 +82,21 @@ class Post
      */
     private $comments;
 
+    /**
+     * @var Collection|Tag[]
```

Comment:
I wouldn't use the `Collection` type here. Client code shouldn't need to care about the actual type (it's sufficient to know that there are traversable tags).